ocp_resources/datavolume.py

```diff
@@ -183,7 +184,17 @@ def wait_deleted(self, timeout=TIMEOUT):
         super().wait_deleted(timeout=timeout)
         return self.pvc.wait_deleted(timeout=timeout)
 
-    def wait(self, timeout=600):
+    def wait(self, timeout=600, failure_timeout=30):
+        # Avoid waiting for "Succeeded" status if DV's in pending/failed status
+        for sample in TimeoutSampler(
+            wait_timeout=failure_timeout,
+            sleep=1,
+            func=lambda: self.instance.get("items", [""])[0].status
+            in [self.Status.PENDING, self.Status.FAILED],
```

Comment:
Could we move this to its own method?
`self.wait_for_status_not(status=[self.Status.PENDING, self.Status.FAILED], timeout=failure_timeout)`

Reply:
done